### ISSUE TITLE
Better support for Markdown package

### ISSUE THREAD

**Issue by jabcross** [overleaf/overleaf#1029]
I really enjoy using markdown for my projects. When I discovered the `markdown` [package](https://www.overleaf.com/learn/how-to/Writing_Markdown_in_LaTeX_Documents#Including_code_in_markdown) I thought I'd be set! The editor even has OK syntax highlighting.

The only thing that keeps me from adopting it outright is the poor support for the "goto location" feature. It always goes to the `\markdownInput` command or similar and skips the source file outright. I'd say this qualifies as a bug.

## Steps to Reproduce
1. Write some markdown using the `markdown` package
2. Double-click some word in the PDF or click the "goto PDF location" button

## Expected Behaviour
Editor should open the file and go to the line containing the highlighted word, even if it is in a markdown context

## Observed Behaviour
Editor only goes as far as a .tex file, and the `\begin{markdown}` or `\markdownInput` command

## Context
Writing a large document separated into several files.

## Technical Info
Using `\usepackage[fencedCode, hybrid]{markdown}`.

* Browser Name and version: Firefox 101
* Operating System and version (desktop or mobile): Linux desktop

**Comment by FramingApp** [comment]
Agreed. The MD package adds a realy easy and neat way for people to transition to latex without having to learn the heavy syntax.
The "File Outline" window is also blank, which is likely a simialr problem.
It would also be nice if the rich text editor supported MD syntax better.
